cloud_governance/policy/aws/cost_over_usage.py

```diff
@@ -9,6 +9,11 @@
 
 
 class CostOverUsage(ElasticUpload):
+    """"
+    Fetches the last 7 days cost_explorer data from the ElasticSearch,
+    then aggregate the data. Calculate cost of user data, if the
+    user data is exceeded maximum cost threshold send alert mail to the User.
+    """
 
```

Comment:
This class checks if the user data is exceeded maximum cost threshold send alert mail to the User by fetching the last 7 days cost_explorer data from the ElasticSearch then aggregate the data.